### PR TITLE
Avoid delayed displaying of sources in source picker

### PR DESCRIPTION
**Pull request by SimonBrandner**
Hopefully explained by the comment

### CODE REVIEW

File: src/components/views/elements/DesktopCapturerSourcePicker.tsx [hunk 1]
```diff
@@ -85,6 +85,15 @@ export default class DesktopCapturerSourcePicker extends React.Component<
...
 
     componentDidMount() {
+        // setInterval() first waits and then executes, therefore
+        // we call getDesktopCapturerSources() here without any delay.
+        // Otherwise the dialog would be left empty for some time.
+        getDesktopCapturerSources().then((result) => {
+            this.setState({
+                sources: result,
+            });
+        });
```
Comment (t3chguy) [inline src/components/views/elements/DesktopCapturerSourcePicker.tsx:95]: can we use async/await here just to match the style of what it looks like within the setInterval too, `componentDidMount` should be fine to be async

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/elements/DesktopCapturerSourcePicker.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/components/views/elements/DesktopCapturerSourcePicker.tsx:95]: I didn't think that was possible. I'll do that

Comment (t3chguy) [reply src/components/views/elements/DesktopCapturerSourcePicker.tsx]: I sure hope it is :P
image

**Comment by SimonBrandner** [comment]
@t3chguy Thanks for the quick review :smile:

**Pull request merged**